dist/index.css

```diff
@@ -1 +1 @@
-.shop__button__heycater_app{font-family:Arial,sans-serif;position:fixed;cursor:pointer}.heycater__widget_placement__bottom{bottom:10px;right:10px}.heycater__widget_placement__middle{bottom:50%;right:10px}.heycater__widget_placement__top{top:10px;right:10px}.heycater__widget_container,.heycater__widget_cta{display:flex;align-items:center;justify-content:center;flex-direction:row;cursor:pointer}.heycater__widget_cta{width:80px;height:80px;border-radius:100px;border:2px solid #e86202;background-color:#e86202;color:#fff;transition:width .7s;box-shadow:2px 2px rgba(0,0,0,.1)}@media (max-width:600px){.heycater__widget_cta{width:40px;height:40px}}.heycater__widget_cta .heycater__widget_button-text{opacity:1;transition:display .5s;width:100%;left:0;display:none;white-space:nowrap;font-size:16px;cursor:pointer}.heycater__widget_cta:hover{width:180px;transition:width .7s;cursor:pointer}.heycater__widget_cart{margin-top:12px;margin-left:3px;cursor:pointer}.heycater__widget_cta:hover .heycater__widget_cart{display:none;cursor:pointer}.heycater__widget_cta:hover .heycater__widget_button-text{opacity:1;display:flex;align-items:center;transition:opacity .5s;cursor:pointer}
+.shop__button__heycater_app{font-family:Arial,sans-serif;position:fixed;cursor:pointer}.heycater__widget_placement__bottom{bottom:10px;right:10px}.heycater__widget_placement__middle{bottom:50%;right:10px}.heycater__widget_placement__top{top:10px;right:10px}.heycater__widget_container,.heycater__widget_cta{display:flex;align-items:center;justify-content:center;flex-direction:row;cursor:pointer}.heycater__widget_cta{width:80px;height:80px;border-radius:100px;border:2px solid #e86202;background-color:#e86202;color:#fff;transition:width .7s;box-shadow:2px 2px rgba(0,0,0,.1)}@media (max-width:600px){.heycater__widget_cta{width:40px;height:40px}}.heycater__widget_cta .heycater__widget_button-text{opacity:1;transition:display .5s;width:100%;left:0;display:none;white-space:nowrap;font-size:16px;cursor:pointer}.heycater__widget_cta:hover{width:180px;transition:width .7s;cursor:pointer}.heycater__widget_cart{margin-top:12px;margin-left:3px;cursor:pointer}.heycater__widget_cta:hover .heycater__widget_cart{display:none;cursor:pointer}.heycater__widget_cta:hover .heycater__widget_button-text{opacity:1;display:flex;align-items:center;transition:opacity .5s;cursor:pointer}
```

Comment:
@javiersuazo any chance we can move css into js file, so widget could load faster in non-render-blocking way?

Reply:
We can, I would address it on a different ticket tho. We either upgrade to parcel 2 or we use a library like style components. Just to not increase complexity on this ticket.